packages/wpcom-proxy-request/src/index.d.ts

```diff
@@ -0,0 +1,20 @@
+export interface WpcomClientCredentials {
+	client_id: string;
+	client_secret: string;
+}
```

Comment:
I think these are specific to a certain endpoint or endpoints - signup perhaps? I don't think they're a general concept we should contain in this library.

cc: @andrewserong @p-jackson

Reply:
Moved to `packages/data-stores/src/shared-types.ts` per feeff98.